### PR TITLE
add os.read_file()

### PR DESCRIPTION
**Pull request by ripienaar**

### CODE REVIEW

**Comment by ripienaar** [comment]
used `readfile` not `read_file` since you already have `readlink`

**Comment by d5** [comment]
It's not a strict rule, but, I tried to respect Go equivalent function's Camelcase. For example. `readlink` was `os.Readlink`. So I think it should be `read_file` as it's matching `ioutil.ReadFile`.

File: docs/stdlib-os.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ os := import("os")
 - `lookup_env(key string) => string/false`: retrieves the value of the environment variable named by the key.
 - `mkdir(name string, perm int) => error `: creates a new directory with the specified name and permission bits (before umask).
 - `mkdir_all(name string, perm int) => error `: creates a directory named path, along with any necessary parents, and returns nil, or else returns an error.
+- `readfile(name string) => array(byte)/error `: reads the contents of a file into a byte array
```
Comment (d5) [inline docs/stdlib-os.md]: should be `=> [byte]/error` to make it consistent with other docs.

**Comment by d5** [comment]
It's not a strict rule, but, I tried to respect Go equivalent function's Camelcase. For example. `readlink` was `os.Readlink`. So I think it should be `read_file` as it's matching `ioutil.ReadFile`.

**Comment by ripienaar** [comment]
update

**Comment by ripienaar** [comment]
travis needs to be reran was an environmental failure

**Pull request merged**